src/App.js

```diff
@@ -25,10 +25,14 @@ class App extends React.Component {
         completed: false
       }
```

Comment:
- Timi, great job completing the TodoApp. Your app is fully functional 
- You are able to search and clear any todo. Nice job on that
-  Your code is clean and You are properly splitting up the code into multiple components.